packages/create-project/commands/init/utils.js

```diff
@@ -1,25 +1,30 @@
 const projects = {
   '@neutrinojs/airbnb': {
+    type: 'linting',
```

Comment:
What do you think if we store the types in a constant.

```
const TYPES = {
  LINTING: 'linting',
  PROJECT: 'project',
  TESTING: 'testing'
}
```

Reply:
👍